src/Symfony/Bridge/Twig/Mime/Renderer.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Mime;
+
+use League\HTMLToMarkdown\HtmlConverter;
+use Twig\Environment;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 4.3
+ */
+final class Renderer
```

Comment:
Where is the class used? Or where do we plan to use it?

Reply:
I thought you were at my keynote, listening carefully :) Everything is explained in the presentation linked in the description of the PR.

Reply:
my core point is related to the design of the class :)
if it's final and has no interface, then type-hinting for it means creating strong coupling, while it could be usefull to provide an alternative implementation (eg another html-to-txt - I used to use `w3m` for that job and it does it nicely)